scripts/download.pl

```diff
@@ -70,27 +70,50 @@ ()
 	return undef;
 }
 
-sub download_cmd($) {
+sub download_cmd {
 	my $url = shift;
 	my $have_curl = 0;
-
-	if (open CURL, "curl --version 2>/dev/null |") {
+	my $have_aria2c = 0;
+	my $fn = shift;
+	my @mrs = @_;
+	my $murl = "'$url'";
+
+	my @chArray = ('a'..'z', 'A'..'Z', 0..9);
+	my $rfn = join '', map{ $chArray[int rand @chArray] } 0..9;
```

Comment:
Should we append the filename to the random string? Also wonder if a better naming would be filename_token or something like that?

Reply:
I think random string is enough